server/views/pages/support/offenderDetail.njk

```diff
@@ -201,6 +201,10 @@
                         <th scope="row" class="govuk-table__header">Recall</th>
                         <td class="govuk-table__cell">{{ prisonerDetail.recall }}</td>
                     </tr>
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">IS-91 status</th>
+                        <td class="govuk-table__cell">{{ "Yes" if is91Status else "No" }}</td>
+                    </tr>
```

Comment:
Currently displays as:
<img width="859" alt="Screenshot 2025-02-14 at 15 16 11" src="https://github.com/user-attachments/assets/71fbab3f-f493-44b1-a752-b73a36c6d40e" />

Maybe makes more sense to set the row title to `IS-91` instead of `IS-91 status`? If the status is either Yes or No 🤔

Reply:
Possibly something like `Is IS91 case?` would be more descriptive? It doesn't matter much though as this is an internally-used page, so it doesn't have to be perfect!